src/ethereum/constantinople/utils/address.py

```diff
@@ -59,3 +59,32 @@ def compute_contract_address(address: Address, nonce: Uint) -> Address:
     canonical_address = computed_address[-20:]
     padded_address = canonical_address.rjust(20, b"\x00")
     return Address(padded_address)
+
+
+def compute_create2_contract_address(
+    address: Address, salt: Bytes32, call_data: bytearray
+) -> Address:
+    """
+    Computes address of the new account that needs to be created, which is
+    based on the sender address, salt and the call data as well.
+
+    Parameters
+    ----------
+    address :
+        The address of the account that wants to create the new account.
+    salt :
+        Address generation salt.
+    call_data :
+        The code of the new account which is to be created.
+
+    Returns
+    -------
+    address: `ethereum.constantinople.eth_types.Address`
+        The computed address of the new account.
+    """
+    preimage = b"\xff" + address + salt + keccak256(call_data)
+    computed_address = keccak256(preimage)
+    canonical_address = computed_address[-20:]
+    padded_address = canonical_address.rjust(20, b"\x00")
```

Comment:
Can you use [left_pad_zero_bytes](https://github.com/ethereum/execution-specs/blob/db01c78535df658e3a1e9eaa65a890cd501171a8/src/ethereum/utils/byte.py#L17-L33) utility method instead?